easybuild/easyblocks/n/numpy.py

```diff
@@ -48,6 +48,21 @@
 from distutils.version import LooseVersion
 
 
+def parse_numpy_test_suite_output(full_testsuite_output):
+    # Parse the last (non-empty) line of the testcmd_output which contains the summary from the test suite
+    testsuite_summary = [i for i in full_testsuite_output.split('\n') if i != ''][-1]
```

Comment:
I think you could use `testsuite_summary = full_testsuite_output.strip().splitlines()[-1]`

Reply:
That works, thanks.